src/components/students-in-categories/StudentsInCategoriesChart.tsx

```diff
@@ -0,0 +1,70 @@
+import { Doughnut } from 'react-chartjs-2'
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js'
+import { Box, Typography } from '@mui/material'
+import { styles } from './StudentsInCategories.styles'
+import { useTranslation } from 'react-i18next'
+
+ChartJS.register(ArcElement, Tooltip, Legend)
+
+const categories = [
+  { label: 'Languages', value: 75, color: '#79B260' },
+  { label: 'Mathematics', value: 12, color: '#FFD166' },
+  { label: 'History', value: 8, color: '#EF476F' },
+  { label: 'Other (+3)', value: 5, color: '#6C757D' }
+]
+
+const data = {
+  labels: categories.map((category) => category.label),
+  datasets: [
+    {
+      data: categories.map((category) => category.value),
+      backgroundColor: categories.map((category) => category.color),
+      hoverBackgroundColor: categories.map((category) => category.color),
+      borderWidth: 1,
+      cutout: '70%',
+      rotation: 180
+    }
+  ]
+}
+
+const options = {
```

Comment:
this probably also should be relocated to consts

Reply:
This object directly belongs to component properties, but not a hardcoded value which is used as input data for the component. That's why it is better to leave it at the same place as the component

Reply:
I still don't understand what is the difference between these hardcoded values and other hardcoded values that we store in separate files with constants